drivers/scheduler/k8s/k8s.go

```diff
@@ -90,9 +90,10 @@ const (
 	// DefaultTimeout default timeout
 	DefaultTimeout = 2 * time.Minute
 
-	resizeSupportedAnnotationKey  = "torpedo.io/resize-supported"
-	autopilotEnabledAnnotationKey = "torpedo.io/autopilot-enabled"
-	specObjAppWorkloadSizeEnvVar  = "SIZE"
+	resizeSupportedAnnotationKey              = "torpedo.io/resize-supported"
+	autopilotEnabledAnnotationKey             = "torpedo.io/autopilot-enabled"
+	rollingDeleteStrategyEnabledAnnotationKey = "torpedo.io/rolling-delete-strategy-enabled"
```

Comment:
instead, call the key `"torpedo.io/delete-strategy"` and one of the supported values it takes is `rolling`. 

This will allow us to have other delete strategies.

Reply:
fixed